cppcache/integration-test/ThinClientListenerWriter.hpp

```diff
@@ -289,23 +299,23 @@ DUNIT_TASK_DEFINITION(CLIENT2, SetupClient2withCachingEnabled_Pooled_Locator)
     setCacheWriter(myRegNames[2], regWriter);
 
     // create subregion
-   auto exmpRegptr = getHelper()->getRegion(myRegNames[2]);
-   auto lattribPtr = exmpRegptr->getAttributes();
-   auto subregPtr1 = exmpRegptr->createSubregion(myRegNames[3], lattribPtr);
-   auto subregPtr2 = exmpRegptr->createSubregion(myRegNames[4], lattribPtr);
+    auto exmpRegptr = getHelper()->getRegion(myRegNames[2]);
+    auto lattribPtr = exmpRegptr->getAttributes();
+    auto subregPtr1 = exmpRegptr->createSubregion(myRegNames[3], lattribPtr);
+    auto subregPtr2 = exmpRegptr->createSubregion(myRegNames[4], lattribPtr);
 
-   LOGINFO(
-       "CLIENT2 SetupClient2withCachingEnabled_Pooled_Locator:: subRegions "
-       "created successfully");
+    LOGINFO(
+        "CLIENT2 SetupClient2withCachingEnabled_Pooled_Locator:: subRegions "
+        "created successfully");
 
-   // Attach Listener to subRegion
-   // Attache Listener
+    // Attach Listener to subRegion
+    // Attache Listener
```

Comment:
These comments add no value, let's get rid of em

Reply:
They don't but this was an auto format change.

Reply:
Guess we'll get to this when we refactor all the test code anyway...